pkg/apis/dashboard/v0alpha1/search.go

```diff
@@ -51,19 +51,9 @@ type SortableField struct {
 	Type    string `json:"type,omitempty"` // string or number
 }
 
-// Dashboard or folder hit
-// +enum
-type HitKind string
-
-// PluginType values
-const (
-	HitTypeDash   HitKind = "dashboard"
-	HitTypeFolder HitKind = "folder"
-)
-
 type DashboardHit struct {
 	// Dashboard or folder
-	Kind HitKind `json:"kind"`
+	Resource string `json:"resource"` // dashboards | folders
```

Comment:
rather than introducing a new thing `HitKind`, we can use the resource value directly